### DISCUSSION TITLE
[Weekly Curation Report]: August 28 - September 1

### DISCUSSION THREAD

**Discussion by mkutmon** [wikipathways/wikipathways-help#64] [curation-reports]
### Recent changes

Some curation work by @khanspers (literature references and description + some curation) and @mkutmon (ontology tags)

### Under construction

No pathways tagged >30 days ago.

### New pathways

2 new pathways
- [WP5413](https://classic.wikipathways.org/index.php/Pathway:WP5413)
  - very large IL24 pathway (NetPath style but lines seem to be mainly connected!)
  - indicated to be manually curated
  - larger subpathways combined in one (not connected) - suggestion to maybe split it up into four pathways?
  - indication of gene expression changes as color - not clear based on which dataset. Will send a welcome message and start a discussion with the new user. 
- [WP5414](https://classic.wikipathways.org/index.php/Pathway:WP5414) - PFOCR-based conversion from https://pfocr.wikipathways.org/figures/PMC6090955__13045_2018_644_Fig1_HTML.html

### Deleted Pathways

- @khanspers, okay to delete? [WP5404](https://classic.wikipathways.org/index.php/Pathway:WP5404)

### Additional notes

- No stub pathways.
- Search works

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @mkutmon! 

[WP5404](https://classic.wikipathways.org/index.php/Pathway:WP5404) was used as a test pathway for a [PathVisio bug](https://github.com/PathVisio/pathvisio/issues/131), but the gpml is attached to that issue so the pathway can indeed be deleted (done).
